great_expectations/expectations/expectation.py

```diff
@@ -2621,3 +2608,29 @@ def _format_map_output(
         return return_obj
 
     raise ValueError(f"Unknown result_format {result_format['result_format']}.")
+
+
+def _validate_mostly_config(configuration: Optional[ExpectationConfiguration]) -> None:
+    """
+    Validates "mostly" in ExpectationConfiguration is a number if it exists.
+
+    Args:
+        configuration: The ExpectationConfiguration to be validated
+
+    Raises:
+        AssertionError: An error is mostly exists in the configuration but is not between 0 and 1.
+    """
+    if "mostly" in configuration.kwargs:
+        mostly = configuration.kwargs["mostly"]
+        assert isinstance(
+            mostly, (int, float)
+        ), "'mostly' parameter must be an integer or float"
+        assert 0 <= mostly <= 1, "'mostly' parameter must be between 0 and 1"
+
+
+def _mostly_success(
```

Comment:
Initially I wanted to generalize this computation for the 3 expectations that use `mostly`. However, the columns they aren't identical and it would have taken a big change to do that so I opted for this function for the time being.